src/components/form-access/FormAccess.tsx

```diff
@@ -107,7 +107,8 @@ export const FormAccess: FunctionComponent<FormAccessProps> = ({
     });
   };
 
-  const isDisabled = !hasAccess(role) && !fineGrainedAccess;
+  let isDisabled = !hasAccess(role);
+  if (fineGrainedAccess !== undefined) isDisabled = !fineGrainedAccess;
```

Comment:
This seems like it could be a tenary.
```suggestion
  const  isDisabled = fineGrainedAccess !== undefined ? !ineGrainedAccess : !hasAccess(role);
```